Update allinone.css

CilginSinek opened the pull request:
Add mobile css and remove unnecessary margins

CilginSinek committed:
Update allinone.css

Add mobile css and remove unnecessary margins

CrawLeyYou commented on static/allinone.css:
```diff
@@ -64,10 +64,12 @@ h4 {
     list-style: none;
-    margin: 10px;
+    /* margin: 10px; */
+    /* margin-bottom: 30px; */
```
If you delete this margins it corrupts the navigation bar 
Your Change:
![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/52cb30f2-109b-4355-bd71-61139bf2ff65)
Stable:
![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/8c067ca9-8dcc-4bdb-89ac-0abfdda1dbac)

CrawLeyYou commented on static/allinone.css:
```diff
@@ -246,3 +248,9 @@ margin-right: auto;
 list-style: none;
 }
+/* for mobile */
+@media screen and (max-width: 430px) {
```
This change doesnt change anything 
![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/eac6f441-4297-4451-a906-f341f12123b1)


![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/3d82cd8b-85d9-4d1a-8276-fceade374e46)

CilginSinek committed:
Update allinone.css

media kullandığımdan dolayı o margine ihtiyacım varmış :p, section elementini yanlış yazmışım

CrawLeyYou commented on static/allinone.css:
```diff
@@ -249,8 +248,8 @@ font-size: 12px;
+    section {
+        margin-top: 100px;
+    }
 }
```
Bu sefer güzel olmuş en azından 3 çizgi yapana kadar kullanabilirim bunu

![image](https://github.com/CrawLeyYou/CrawLeyYou.github.io/assets/60201017/0e64211e-7a75-4699-b9fa-393cc3dc376f)
